src/Juvix/Interpreter/InteractionNet/Nets/Default.hs

```diff
@@ -587,8 +636,28 @@ curry1 (nodeF, numNode) (p, numPrim) =
       deleteRewire [numNode, numPrim] [curr]
       pure $ True
 
+curryPrim1 ∷
+  InfoNetwork net (Lang primVal) m ⇒
+  (primVal → Maybe primVal, Node) →
+  (primVal, Node) →
+  m Bool
+curryPrim1 (nodeF, numNode) (p, numPrim) =
+  case nodeF p of
+    Nothing → pure False
+    Just x → do
+      let node = PrimVal x
+      incGraphSizeStep (- 1)
+      curr ← newNode (Primar node)
+      let currNode = RELAuxiliary0
+            { node = curr,
+              primary = ReLink numNode Aux1
+            }
+      linkAll currNode
+      deleteRewire [numNode, numPrim] [curr]
+      pure $ True
+
```

Comment:
Same for this vs curry1

Reply:
Ditto on the answer.

Reply:
Noted in https://github.com/cryptiumlabs/juvix/issues/184.